llap-server/src/java/org/apache/hadoop/hive/llap/counters/WmFragmentCounters.java

```diff
@@ -119,6 +123,21 @@ public void dumpToTezCounters(TezCounters tezCounters, boolean isLast) {
     for (int i = 0; i < fixedCounters.length(); ++i) {
       tezCounters.findCounter(LlapWmCounters.values()[i]).setValue(fixedCounters.get(i));
     }
+
+    // add queue and runtime (together with task count) on a "per daemon" level
+    // to the Tez counters.
+    if (addTaskTimeCounters) {
```

Comment:
Do we expect to calculate this values to be so time consuming to merit it's own configuration value? Having too many configs had been a continuous complaint from our users, so lately I try to avoid them if not necessary.
Or we do this because of backward compatibility?

Reply:
I wasn't too concerned about the costs of generating the output but about the length. This is essentially adding three counters (and therefore 3 text lines) per LLAP node to the output of every query. If we have very large LLAP clusters, this might be very disturbing just from a consumability perspective.
Also: These information are more for our own optimization efforts than for use by end users or customers. So, I wouldn't want to bother "normal" users with this output.

Reply:
Makes sense. Thanks!